js/bybit.js

```diff
@@ -3114,13 +4148,19 @@ module.exports = class bybit extends Exchange {
         const market = this.market (symbol);
         symbol = market['symbol'];
         const isUsdcSettled = (market['settle'] === 'USDC');
+        let enableUnifiedMargin = undefined;
+        [ enableUnifiedMargin, params ] = this.handleOptionAndParamsValue (params, 'createOrder', 'enableUnifiedMargin', false);
+        const isV3 = (this.version === 'v3');
         if (market['spot']) {
             return await this.createSpotOrder (symbol, type, side, amount, price, params);
+        } else if (enableUnifiedMargin) {
+            return await this.createUnifiedMarginOrder (symbol, type, side, amount, price, params);
         } else if (isUsdcSettled) {
             return await this.createUsdcOrder (symbol, type, side, amount, price, params);
-        } else {
-            return await this.createContractOrder (symbol, type, side, amount, price, params);
+        } else if (isV3) {
```

Comment:
@sc0Vu shouldn't we check `if v3` before `is USDC` ? afaik in v3 USDC orders go to the same endpoint as the other derivatives

Reply:
As I saw in testnet contract account, I think contract v3 is for usdt contract.